addons/hr_expense/tests/test_expenses.py

```diff
@@ -1031,3 +1031,39 @@ def test_foreign_currencies_total(self):
         }])
         expense_state = Expense.get_expense_dashboard()
         self.assertEqual(expense_state['to_submit']['amount'], 3000.00)
+
+    def test_payment_register_bank_from_expense_reimbursed_to_employee(self):
+        """
+        Test that creating an expense to be paid to an employee having a commercial partner (the company listed in the
+        employee's contact) will have the employee's bank account in the register payment wizard.
+        """
+        # Set bank account in employee.
+        self.expense_employee.bank_account_id = self.env['res.partner.bank'].create({
+            'acc_number': 'BE32707171912447',
+            'partner_id': self.expense_employee.work_contact_id.id,
+            'acc_type': 'bank',
+        })
+        # Set bank account in company.
+        self.env.company.partner_id.bank_ids = self.env['res.partner.bank'].create({
+            'acc_number': 'BE457268179587463',
+            'partner_id': self.env.company.id,
```

Comment:
```suggestion
            'partner_id': self.env.company.partner_id.id,
```
Fails in no-demo build